libwild/src/string_merging.rs

```diff
@@ -331,20 +343,22 @@ impl<'data> MergedStringsSection<'data> {
             return Ok(());
         }
 
-        let num_threads = args.string_merging_threads().get();
-
-        let mut resources =
-            create_split_resources(&mut self.string_offsets, input_sections, num_threads);
-
-        // Spawn a task for each thread.
-        rayon::scope(|s| {
-            for _ in 0..num_threads {
-                s.spawn(|_| {
-                    if let Err(error) = do_splitting_work(&resources, reuse_pool) {
-                        let _ = resources.errors.push(error);
-                    }
-                });
-            }
+        let num_threads = rayon::current_num_threads();
```

Comment:
Leaving it to the default number of threads seems to be a rather big hit beyond 20 threads.
`wild-pwmu` is this PR and `wild-pwmu2` is this PR with a hack:

```suggestion
        let num_threads = args.numeric_experiment(Experiment::MergeStringCapMultiplier, 80) as usize
            / 10;
```
https://gist.github.com/mati865/e90546d1e9d31b880db57a92e0de84ce